bundler/lib/bundler/checksum.rb

```diff
@@ -161,26 +165,17 @@ class Store
 
       def initialize
         @store = {}
-      end
-
-      def initialize_copy(other)
```

Comment:
Why are you removing this?

Reply:
It's never used, but if you think there's value to it. I couldn't think of a case where we'd dup it legitimately and there's no test for it.

Reply:
In actuality, when I was adding the mutex I had to duplicate the mutex into here, then also lock the copy process and it seemed excessive for something that we don't use.

Reply:
ah if it is never used then we're good, I remember it was used at some point